internal/pkg/template/workload.go

```diff
@@ -117,6 +118,12 @@ type Volume struct {
 	IAM           *string // ENABLED or DISABLED
 }
 
+// AccessPointInfo holds information about how to create Copilot-managed access points
+type AccessPointInfo struct {
+	Uid *string
+	Gid *string
```

Comment:
Can we rename the fields here and in the template to match https://github.com/golang/go/wiki/CodeReviewComments#initialisms?
```suggestion
	UID *string
	GID *string
```

Reply:
I've adopted a more thoughtful struct name for this, which will be easier to extract from the manifest and inject into the template, and addressed the capitalization.